qiskit/circuit/library/standard_gates/u1.py

```diff
@@ -177,6 +177,22 @@ class CU1Gate(ControlledGate):
     This is a diagonal and symmetric gate that induces a
     phase on the state of the target qubit, depending on the control state.
 
+    .. warning::
+
+       This gate is deprecated. Instead, the following replacement should be used
```

Comment:
It would be nice to link the `CPhaseGate` as clickable link. You can do that with the Sphinx `:class:` identifier:
```suggestion
       This gate is deprecated. Instead, the :class:`.CPhaseGate` should be used
```